### PR TITLE
Mobile and Touch Events for Featured Tags/ Viewing Options

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Adds touch event for featured tags https://www.pivotaltracker.com/story/show/148441077
- Fix and adds filter Icon for Viewing Options https://www.pivotaltracker.com/story/show/149353761

### CODE REVIEW

File: plugins/talk-plugin-featured-comments/client/components/Tag.js [hunk 1]
```diff
@@ -15,6 +16,20 @@ export default class Tag extends React.Component {
 
   }
 
+  componentDidMount() {
+    if (bowser.mobile) {
```
Comment (alinhle) [inline plugins/talk-plugin-featured-comments/client/components/Tag.js]: Not all touch devices are `mobile`. Is it necessary to make this check?

File: plugins/talk-plugin-featured-comments/client/components/Tag.js [hunk 2]
```diff
@@ -15,6 +16,20 @@ export default class Tag extends React.Component {
...
+      this.tagEl.addEventListener('touchstart', this.showTooltip);
```
Comment (alinhle) [inline plugins/talk-plugin-featured-comments/client/components/Tag.js]: Why you don't use React events? https://facebook.github.io/react/docs/events.html#touch-events

**Comment by kgardnr** [comment]
OMGGGG I love the viewing options icon @IAmSamHankins & @okbel!!

**Review by kgardnr — APPROVED** [review]
:100:

**Pull request merged**